bears/natural_language/LanguageToolBear.py

```diff
@@ -33,16 +33,16 @@ def check_prerequisites(cls):
             except ImportError:  # pragma: no cover
                 return 'Please install the `language-check` pip package.'
 
-    @deprecate_settings(language='locale')
+    @deprecate_settings(natural_language='locale')
```

Comment:
I think we'll also want to have tests that verify that all deprecated variants work properly